src/From.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ArchTech\Enums;
+
+trait From
+{
+    /**
+     * Gets the Enum by index for "Pure" enums.
+     *
+     * This will not override the `from()` method on BackedEnums
+     *
+     * @throws \ValueError
+     */
+    static public function from(int $case): static
+    {
+        return static::cases()[$case] ?? throw new \ValueError($case . ' is not a valid unit for enum "' . get_called_class() . '"');
```

Comment:
- Import the `ValueError`
- Is `get_called_class()` any different from `static::class`?